shared/services/rocketpool/client.go

```diff
@@ -1084,7 +1079,7 @@ func (c *Client) compose(composeFiles []string, args string) (string, error) {
 	}
 
 	// Return command
-	return fmt.Sprintf("%s docker compose --project-directory %s %s %s", strings.Join(env, " "), shellescape.Quote(expandedConfigPath), strings.Join(composeFileFlags, " "), args), nil
```

Comment:
I can't think of any good reason to pass the env vars to `docker compose` after we already formatted them into the tmpl files